salt/states/zabbix_host.py

```diff
@@ -184,6 +184,13 @@ def _interface_format(interfaces_data):
         inventory = kwargs['inventory']
     if inventory is None:
         inventory = {}
+        
+    new_visible_name = ''
```

Comment:
you should use:
new_visible_name = kwargs.get('visible_name', '')
because what you do is an anti-pattern in python:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_get_to_return_a_default_value_from_a_dictionary.html

Reply:
Fair, and fixed - though I would note that there is literally no use of kwargs.get anywhere else in this file. I was simply following the established pattern of the file.

Reply:
Maybe that was declared an anti-pattern after. :)

Reply:
I'll have to grab a Python environment for doing this in the future, apologies for all the commits. Doing this while overtired isn't a great idea.